src/entities/commit/commitEntity.js

```diff
@@ -17,8 +18,16 @@ import scoreTestCommitType from "./services/scoreTestCommitType";
  * qualityScore: number,
  * }}
  */
-const createCommitEntity = ({ type, sha, url, author, message }) => {
+const createCommitEntity = ({
+  formatStyle,
+  type,
+  sha,
+  url,
+  author,
+  message,
+}) => {
   return {
+    formatStyle: formatStyle,
```

Comment:
key와 value가 같은 이름이라 짧은 표기법으로 표현할 수 있을 것 같습니다~
```suggestion
  return {
    formatStyle,
```
[Object initializer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)

Reply:
좋은 생각이네요! 아래도 짧은 표기법으로 바꾸도록 하겠습니다!